app/src/main/java/ro/code4/monitorizarevot/repositories/Repository.kt

```diff
@@ -125,15 +135,15 @@ class Repository : KoinComponent {
     fun getForms(): Observable<Unit> {
 
         val observableDb = db.formDetailsDao().getAllForms().toObservable()
-
         val observableApi = apiInterface.getForms()
 
+        // todo fix this as it does not work as expected. nulls are not treated as return values.
+        //  instead it throws a NullPointerException.
         return Observable.zip(
             observableDb.onErrorReturn { null },
             observableApi.onErrorReturn { null },
```

Comment:
@aniri I think an issue should be created for it.

Reply:
I plan to fix it in a future task. I tried but the idea is to not explode the PR.

Reply:
yep, I'll add a new issue for this after we finish merging this PR